.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
   "files.exclude": {
-    "**/*.bs.js": false
+    "**/*.bs.js": true
```

Comment:
Revert this change

Reply:
Done with https://github.com/winoteam/re-big/pull/2/commits/1eb12a737cc31c5842c2158506ca34e83269aeb3